chainer_chemistry/dataset/parsers/sdf_file_parser.py

```diff
@@ -68,7 +74,9 @@ def parse(self, filepath, return_smiles=False, target_index=None):
                 mol = mol_supplier[int(index)]
 
                 if mol is None:
-                    total_count -= 1
+                    fail_count += 1
```

Comment:
How to consider the case of "mol is None".
Previously, I was dealing this case is "not included in the counting itself".
However, now I feel it is better to consider this case as simply "fail, with counting".

Reply:
Anyway, this will only affect to logging in L153. It does not affect to program input/output behavior.

Reply:
LGTM.
BTW, why did you choice "not included in the counting itself"?
Is there any specific reason to do it?

Reply:
Because when mol is None, this is not even included in preprocessing trial, I thought at the first time it is better to just exclude it.
But now if we want to refer `is_successful` array, i think containing all information is better.

Reply:
I see. Thank you.